src/MetaModels/Attribute/TranslatedUrl/TranslatedUrl.php

```diff
@@ -218,6 +221,9 @@ public function setTranslatedDataFor($values, $language)
         $time   = time();
         $params = array();
         foreach ($values as $id => $value) {
+            if (!count(array_filter((array)$value))) {
```

Comment:
This will break the query which is built above.
You must filter before building the query string.

I suggest creating the `$sql` after determining the parameters.